homeassistant/components/radarr/const.py

```diff
@@ -1,12 +1,11 @@
 """Constants for Radarr."""
 import logging
-
-from typing_extensions import Final
+from typing import Final
 
 DOMAIN: Final = "radarr"
 
 # Defaults
 DEFAULT_NAME = "Radarr"
-DEFAULT_URL = "http://172.17.0.1:7878"
+DEFAULT_URL = "http://127.0.0.1:7878"
```

Comment:
If we want explicit support for the add-on, we can detect if we're in supervisor and have the add-on, and just offer an option to set that up without asking for a URL.

Reply:
Did you want this for the next PR? This PR s not pinned to `dev`.

Reply:
oh this is not going to dev? now I am confused 🤣